cmd/up.go

```diff
@@ -148,3 +161,62 @@ func fillVariables(vars types.Variables) types.Variables {
 	}
 	return vars
 }
+
+func setRunConfigVars(rc *types.RunConfig, varFlags []string) error {
+	var errs []string
+
+	for _, varFlag := range varFlags {
+		tokens := strings.SplitN(varFlag, "=", 2)
+		if len(tokens) < 2 {
+			// We skip any variables that don't have an `=` sign
+			continue
+		}
+		name, value := tokens[0], tokens[1]
+		// Look in RunConfig for variable matching `name`
+		// If we don't find a matching variable, we just skip it
```

Comment:
Done. Here is an example:
```
$ nullstone up --block=test --env=dev --var=service_port=fail --var=unknown=x
Performing workspace command (Org=demo, Block=test, Stack=, Env=dev)

[Warning] The following variables were skipped because they don't exist in the module: unknown

"--var" flags contain invalid values:
    * service_port: expected 'number' - strconv.ParseFloat: parsing "fail": invalid syntax
```